recipes/R/Retail - GradientBoosting/R/applicationEvaluator.R

```diff
@@ -55,42 +60,40 @@ applicationEvaluator <- setRefClass("applicationEvaluator",
         }
       }
       df <- df %>%
-        mutate(store = as.numeric(store)) %>% 
+        mutate(store = as.numeric(store)) %>%
         mutate(date = mdy(date), week = week(date), year = year(date)) %>%
         mutate(new = 1) %>%
         spread(storeType, new, fill = 0) %>%
         mutate(isHoliday = as.integer(isHoliday)) %>%
         mutate(weeklySalesAhead = lead(weeklySales, 45),
                weeklySalesLag = lag(weeklySales, 45),
+               weeklySalesScaled = lead(weeklySalesAhead, 45),
                weeklySalesDiff = (weeklySales - weeklySalesLag) / weeklySalesLag) %>%
         drop_na() %>%
         filter(if(!is.null(timeframe)) {
         date >= as.Date(Sys.time()-as.numeric(timeframe)*60) & date <= as.Date(Sys.time())
         } else {
-        date >= "2012-02-03"  
+        date >= "2012-02-03"
         }) %>%
         select(-date)
-      
-      
+
       #########################################
-      # Retrieve saved model and evaluate
+      # Split Data
       #########################################
-      retrieved_model <- readRDS("model.rds")
-      pred <- predict(retrieved_model, df, n.trees = 100)
-      
-      mape <- mean(abs((df$weeklySalesAhead - pred) / df$weeklySalesAhead))
-      mae <- mean(abs(df$weeklySalesAhead - pred))
-      rmse <- sqrt(mean((df$weeklySalesAhead - pred)^2))
-      
-      insight1 <- list(name = "MAPE", valueType = "number",
-                       value = round(mape, 3))
-      insight2 <- list(name = "MAE", valueType = "number",
-                       value = round(mae, 3))
-      insight3 <- list(name = "RMSE", valueType = "number",
-                       value = round(rmse, 3))
-      metrics <- list(insight1, insight2, insight3)
-      
-      return(metrics)
+      seed = 101
+      if(!is.null(configurationJSON$evaluation.seed)){
+        seed = as.numeric(configurationJSON$evaluation.seed)
+      }
+      trainRatio = 0.8
+      if(!is.null(configurationJSON$evaluation.trainRatio)){
+        train_ratio = as.numeric(configurationJSON$evaluation.trainRatio)
```

Comment:
Should this be, `trainRatio` instead of `train_ratio`?  (I don't see any other references to `train_ratio` in the file.

Reply:
Ouh,yes. I made a typo here. Will correct now.